### DISCUSSION TITLE
longtblr表题后有冒号

### DISCUSSION THREAD

**Discussion by yunzhentian** [obster-y/XJTU-thesis#122] [General]
见例子中“表 2-3: A Long Long Long Long Long Long Long Table”， 学校要求是没有冒号的，特别是要加注释只能用longtblr

**Comment by zl-akk** [comment]
你好，我找到了方法。
在main.tex中，在下图位置中输入该行代码
![屏幕截图 2024-04-28 002341](https://github.com/obster-y/XJTU-thesis/assets/166778814/8b4f5108-0c8f-4723-a96f-df7e32887d7c)

**Reply by obster-y** [reply]
确实是个失误，已经在 `v1.3.9.10` 里修复了
